### PR TITLE
[drivers] Specifies the name of the drivers driver file

### PR DESCRIPTION
**Pull request by CXSforHPU**
## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)
组件中的device命名不规范，需要同步下规范

#### 你的解决方案是什么 (what is your solution)
# Component/drivers/include

* gpt.h  -> dev_gpt.h
* mmc.h ->dev_mmc.h
* mmcsd_core.h ->dev_mmcsd_core.h
* sd.h ->dev_sd.h
* sdio.h ->  dev_sdio.h
* serial.h -> dev_serial.h
* serial_v2.h -> dev_serial_v2.h
* spi.h -> dev_spi.h
* touch.h -> dev_touch.h
* watchdog.h ->dev_watchdog.h

# sdio

## 源文件：

* block_dev.c -> dev_block.c

* gpt.c -> dev_gpt.c

* mmc.c -> dev_mmc.c

* mmcsd_cord.c -> dev_mmcsd_cord.c

* sd.c -> dev_sd.c

* sdio.c -> dev_sdio.c

# serial

## 源文件

* serial_v2.c -> dev_serial_v2.c

* serial.c ->  dev_serial.c

​	

# spi

## 源文件：

* spi_core.c -> dev_spi_core.c

* spi_dev.c -> dev_spi.c

* spi_flash_sfud.c -> dev_spi_flash_sfud.c

* spi_msd.c -> dev_spi_msd.c

* spi_wifi_rw009.c -> dev_wifi_rw009.c

* spi-bit-ops.c -> dev_spi_bit_ops.c
* qspi_core.c -> dev_qspi_core.c
 ## 头文件

* spi_flash_sfud.h -> dev_spi_flash_sfud.h
* spi_flash.h -> dev_spi_flash.h
* spi_msd.h -> dev_spi_msd.h
* spi_wifi_rw009.h -> dev_spi_wifi_rw009.h
* spi-bit-ops.h -> dev_spi_bit_ops.h

# touch

## 源文件

touch.c -> dev_touch.c

# watchdog

## 源文件

watchdog.c -> dev_watchdog.c

# wlan

## 源文件

wlan_cfg.c -> dev_wlan_cfg.c

wlan_cmd.c -> dev_wlan_cmd.c

wlan_dev.c -> dev_wlan.c

wlan_lwip.c -> dev_wlan_lwip.c

wlan_mgnt.c -> dev_wlan_mgnt.c

wlan_prot.c -> dev_wlan_prot.c

wlan_workqueue.c -> dev_wlan_workqueue.c

## 头文件

wlan_cfg.h -> dev_wlan_cfg.h

wlan_dev.h -> dev_wlan.h

wlan_mgnt.h -> dev_wlan_mgnt.h

wlan_prot.h -> dev_wlan_prot.h

wlan_workqueue.h -> dev_wlan_workqueue.h

#### 请提供验证的bsp和config (provide the config and bsp) 

- BSP:

- .config:

- action:

]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [x] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [ ] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [ ] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [ ] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [ ] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 代码是高质量的 Code in this PR is of high quality
- [ ] 已经使用[formatting](https://github.com/mysterywolf/formatting) 等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

**Comment by mysterywolf** [comment]
本PR无需强制对格式做出调整，CI如果格式有报错 请无视以下两个：
Static code analysis / Static code analysis (pull_request)
Check File Format and License / Scan code format and license (pull_request)

**Comment by kurisaW** [comment]
@Rbb666 @mysterywolf 这个PR我已经审过了，我这边没什么问题了，你们帮忙看下哈

File: components/drivers/include/rtdevice.h
Comment (Rbb666) [inline components/drivers/include/rtdevice.h:1]: 17-24行的麻烦也帮忙 改成""的引用吧：https://github.com/RT-Thread/rt-thread/blob/779547d708b3e0d41505e50410b73101fe418582/components/drivers/include/rtdevice.h#L20

**Comment by mysterywolf** [comment]
3个CI报警均可忽略

File: components/drivers/include/rtdevice.h
Comment (mysterywolf) [reply components/drivers/include/rtdevice.h:1]: 这个单独提个PR修改吧

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**